specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.benefits.json

```diff
@@ -0,0 +1,1002 @@
+              "$ref": "common-types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "benefitRecommendationsListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "The list of benefit recommendations.",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/benefitRecommendationModel"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The link (URL) to the next page of results.",
+          "readOnly": true,
+          "format": "uri"
+        }
+      },
+      "description": "Result of listing benefit recommendations."
+    },
+    "benefitRecommendationModel": {
+      "type": "object",
+      "description": "benefit plan recommendation details.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BenefitResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "type": "object",
+          "$ref": "#/definitions/benefitRecommendationProperties"
+        }
+      }
+    },
+    "benefitRecommendationProperties": {
+      "type": "object",
+      "description": "The properties of the benefit plan recommendation.",
+      "discriminator": "scope",
+      "properties": {
+        "firstConsumptionDate": {
+          "type": "string",
+          "description": "The first usage date used for looking back for computing the recommendation.",
+          "readOnly": true
+        },
+        "lastConsumptionDate": {
+          "type": "string",
+          "description": "The last usage date used for looking back for computing the recommendation.",
+          "readOnly": true
+        },
+        "lookBackPeriod": {
+          "type": "string",
+          "description": "The number of days of usage used to look back for computing the recommendation.",
+          "readOnly": true
+        },
+        "totalHours": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The total hours for which the cost is covered. Its equal to number of records in a property 'properties/usage/charges'.",
+          "readOnly": true
+        },
+        "usage": {
+          "type": "object",
+          "description": "The additional details about the usage which are considered while computing benefit recommendations.",
+          "$ref": "#/definitions/recommendationUsageDetails"
+        },
+        "armSkuName": {
+          "description": "ARM SKU name. For example, Standard_D1_V2 for Reservations and Compute_Savings_Plan for SavingsPlan.",
+          "type": "string",
+          "readOnly": true
+        },
+        "term": {
+          "type": "string",
+          "description": "Term period of the benefit. For example, P1Y or P3Y.",
+          "readOnly": true
+        },
```

Comment:
what date formats are accepted?

Reply:
Converted this to enum. Only possible values are 'P1Y' and 'P3Y'